tests/integration/update.bats

```diff
@@ -648,3 +648,29 @@ EOF
 	runc resume test_update
 	[ "$status" -eq 0 ]
 }
+
+@test "runc update replaces devices cgroup program" {
+	[[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
```

Comment:
The test is NOP in rootless, right? Can we have a comment line to explain the expected behavior?

Reply:
It's also no-op for cgroupv1...